tomcat/src/main/java/org/apache/coyote/http11/http/HttpHeaders.java

```diff
@@ -0,0 +1,78 @@
+package org.apache.coyote.http11.http;
+
+import static org.apache.catalina.utils.Parser.removeBlank;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import org.apache.coyote.http11.header.HttpHeader;
+import org.apache.coyote.http11.header.HttpHeaderType;
+
+public class HttpHeaders {
+    private final Map<HttpHeaderType, HttpHeader> headers;
+
+    private HttpHeaders(final Map<HttpHeaderType, HttpHeader> headers) {
+        this.headers = headers;
+    }
+
+    public static HttpHeaders of(final BufferedReader bufferedReader) throws IOException {
+        return new HttpHeaders(readAllHeaders(bufferedReader));
+    }
+
+    public static HttpHeaders of(final HttpHeader... httpHeaders) {
+        final Map<HttpHeaderType, HttpHeader> headers = new LinkedHashMap<>();
+        for (HttpHeader httpHeader : httpHeaders) {
+            headers.put(httpHeader.getHttpHeaderType(), httpHeader);
+        }
+        return new HttpHeaders(headers);
```

Comment:
```suggestion
        final Map<HttpHeaderType, HttpHeader> headers = Arrays.stream(httpHeaders)
                .collect(Collectors.toMap(HttpHeader::getHttpHeaderType,
                        httpHeader -> httpHeader, 
                        (key, value) -> value,
                        LinkedHashMap::new));
        return new HttpHeaders(headers);
```
요건 완 전 취향이긴한데 저는 요렇게도 씁니당!

Reply:
stream 사용 좋네요! 반영했습니다.!